doc/developers/testsuite.html

```diff
@@ -779,12 +779,12 @@
 CTEST_CMAKE_GENERATOR
   - The CMake Generator to use (e.g. "Unix Makefiles", or "Ninja", see
     $ man cmake)
-  - If unspecified the current generator of a configured build directory
-    will be used, otherwise "Unix Makefiles".
+  - If unspecified the generator of a configured build directory will
+    be used, otherwise "Unix Makefiles".
 
 TRACK
-  - The track the test should be submitted to. Defaults to "Experimental".
-    Possible values are:
+  - The track the test should be submitted to. Defaults to
+    "Experimental". Possible values are:
```

Comment:
shall we just list other tracks without description and mention that they are only used to separate results from different testers? For more info see https://github.com/dealii/dealii/wiki/Testing-Infrastructure?